node/src/main/scala/org/bitcoins/node/PeerFinder.scala

```diff
@@ -166,19 +172,21 @@ case class PeerFinder(
     //delete try queue
     _peersToTry.clear()
 
-    _peerData.foreach(_._2.client.close())
+    val closeF = Future.sequence(_peerData.map(_._2.client.map(_.close())))
```

Comment:
nit: Sometimes it is better to just add a new line as it makes things more readable IMO. 

```scala
val closed = _peerData.map(_._2.client.map(_.close()))
val closedF = Future.sequence(closed)
```